app/Community/Policies/TriggerTicketPolicy.php

```diff
@@ -15,7 +16,13 @@ class TriggerTicketPolicy
 
     public function manage(User $user): bool
     {
-        return $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
+        return $user->hasAnyRole([
+            Role::HUB_MANAGER,
+            Role::DEVELOPER_STAFF,
+            Role::DEVELOPER,
+            Role::DEVELOPER_JUNIOR,
+        ])
+            || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
```

Comment:
What's the difference between TicketPolicy and TriggerTicketPolicy?

Reply:
Similar to `Comment` vs `AchievementComment`, `TriggerTicket` is a more specific form of `Ticket`. tickets are meant to be migrated to be polymorphic, where tickets can not only be written for triggers (achievements, leaderboards) but for other resources, too.